src/main/java/it/gov/pagopa/pu/debtpositions/service/DebtPositionTypeOrgServiceImpl.java

```diff
@@ -71,4 +81,22 @@ private void handleOperators(DebtPositionTypeOrg debtPositionTypeOrg, SaveDebtPo
         saveDebtPositionTypeOrgDTO.getEnabledOperators());
     }
   }
+
+  private void validateDebtPositionTypeOrg(DebtPositionTypeOrg debtPositionTypeOrg) {
+      if (debtPositionTypeOrg == null) {
+          throw new ValidationException("DebtPositionTypeOrg must not be null");
+      }
+      if(debtPositionTypeOrg.getDebtPositionTypeOrgId()!=null){
+        DebtPositionTypeOrg dpto = debtPositionTypeOrgRepository.findById(debtPositionTypeOrg.getDebtPositionTypeOrgId())
+                .orElseThrow(()->new NotFoundException("DebtPositionTypeOrg having ID %d not found".formatted(debtPositionTypeOrg.getDebtPositionTypeOrgId())));
+        checkReadOnlyFields(dpto, debtPositionTypeOrg);
+      }
+  }
+
+  private void checkReadOnlyFields(DebtPositionTypeOrg existingDebtPositionTypeOrg, DebtPositionTypeOrg updatedDebtPositionTypeOrg) {
+    List<String> updatedFields = ValidationUtils.checkReadOnlyFields(existingDebtPositionTypeOrg, updatedDebtPositionTypeOrg, debtPositionTypeOrgReadOnlyFields);
```

Comment:
instead of using reflection, let's explicitly use getters, moreover, there is already an utility function used to valuate the change on a field:
it.gov.pagopa.pu.debtpositions.util.Utilities#checkImmutableField
let's use it, which will be updated in order to set the required equality function (eg: Temporal data or BigDecimal value could be not equal, evenIf they represent the same data)